service-catalog/docs-rs/REAMDE.md

```diff
@@ -0,0 +1,15 @@
+# Docs.rs
+
+Docs.rs is a service that hosts documentation of crates.
+
+The docs.rs codebase is available at [rust-lang/docs.rs] and
+it's maintained by the [docs.rs team](https://www.rust-lang.org/governance/teams/dev-tools#team-docs-rs).
+
+The codebase is deployed via the [terraform/docs-rs] module.
+
+The EC2 `docsrs.infra.rust-lang.org` is served by Cloudfront CDN at `docs.rs`.
+
+Docs.rs also uses the `rust-docs-rs` S3 bucket which is served by Cloudfront CDN at `static.docs.rs`.
```

Comment:
the bucket is served through the docs.rs webserver itself, and _additionally_ through `static.docs.rs` 

( we are rewriting the doc pages before serving them)